src/integrationTest/java/uk/gov/hmcts/reform/managecase/api/controller/CaseAssignmentControllerIT.java

```diff
@@ -67,8 +67,8 @@
     private static final String ANOTHER_USER = "vcd345cvs-816a-4eea-b714-6654d022fcef";
     private static final String CASE_ID = "1588234985453946";
     private static final String CASE_ID2 = "1598630369818638";
-    private static final String ORG_POLICY_ROLE = "caseworker-probate";
-    private static final String ORG_POLICY_ROLE2 = "caseworker-probate2";
+    private static final String IDAM_USER_ROLE = "caseworker-probate-solicitor";
+    private static final String IDAM_USER_ROLE2 = "caseworker-probate2-solicitor";
```

Comment:
Based on the usage I can see in this PR these constants refer to **Case Roles** not **IDAM User Roles**.

The original `ORG_POLICY_ROLE` name was not that bad as the case roles are stored in the organisation policy.  The issue raised in https://github.com/hmcts/aac-manage-case-assignment/pull/223#discussion_r567685259 relate to the example values which are following the wrong naming convention and therefore may be confused with IDAM roles.

try something like 
```
private static final String ORG_POLICY_ROLE = "[CaseRole1]";
private static final String ORG_POLICY_ROLE2 = "[CaseRole2]";
```